test/javascripts/acceptance/admin-users-list-test.js.es6

```diff
@@ -9,6 +9,50 @@ QUnit.test("lists users", async assert => {
   assert.ok(!exists(".user:eq(0) .email small"), "escapes email");
 });
 
+QUnit.test("sorts users", async assert => {
+  await visit("/admin/users/list/active");
+
+  assert.ok(exists(".users-list .user"));
+
+  await click(".users-list .sortable:nth-child(1)");
+
+  assert.ok(
+    find(".users-list .user:nth-child(1) .username")
+      .text()
+      .includes("eviltrout")
```

Comment:
It is a good practice to include a message about what the test is expecting. Otherwise, it won't be clear which test fails when it happens.